public/js/hsp.js

```diff
@@ -35,12 +35,12 @@ export default class HSP extends React.Component {
   render() {
     return (
       <div
-        className="hsp pt-px pr-0 pl-5 pb-0.5"
+        className="hsp pt-px pb-5"
         id={this.domID()}
         ref={this.hspRef}
         data-parent-hit={this.hitDOM_ID()}
       >
-        <p className="m-0 p-0 rounded-none border-0 bg-inherit whitespace-pre-wrap break-keep text-sm text-gray-700 font-semibold tracking-tighter">
+        <p className="m-0 p-0 rounded-none border-0 bg-inherit whitespace-pre-wrap break-keep text-sm text-neutral-500 font-semibold tracking-[0.7px]">
```

Comment:
`tracking-[0.7px]` - same here, I'd rather stick to standard tailwind sizings even though it looks different from the previous version (current main branch). As long as the font sizes are consistent on the same page I'd prefer to stick to Tailwind conventions